### DISCUSSION TITLE
Archery中SQL上线系统自动驳回NoneType' object is not subscriptable如何处理

### DISCUSSION THREAD

**Discussion by averyyu123** [hhyo/Archery#1797] [General]
![image](https://user-images.githubusercontent.com/108975983/186086677-05e435c7-d22e-4843-aada-7057f82aa34b.png)
简单的update语句，一直提示NoneType' object is not subscriptable，explain语法检查未通过！

**Comment by kateflyup** [comment]
请问你解决这个问题了吗？我也莫名其妙碰到了...

**Reply by averyyu123** [reply]
可能安装或者本身的bug问题吧，同事给处理的说是后台debug处理

**Reply by kateflyup** [reply]
请问您是怎么处理的？必须重装解决吗？

**Reply by averyyu123** [reply]
不是重装的，后台好像有个控制台，修复就行，具体我也不知道怎么处理的因为我不是做开发的，同事帮处理的

**Comment by averyyu123** [comment]
不是重装的，后台好像有个控制台，修复就行

**Reply by kateflyup** [reply]
控制台是指django管理那个页面吗，还是指python的控制台？

**Reply by averyyu123** [reply]
明天问问吧，同事都下班了

**Reply by kateflyup** [reply]
好的，感谢！

**Comment by hhyo** [comment]
参考关联问题  https://github.com/hhyo/Archery/issues/1573
